### PR TITLE
Adds a little more brouzouf, but removes confusion about brouzouf

### LINKED ISSUE #20266: "My legs are OK." when targeting limbs
(WEB REPORT BY: xevonix REMOTE: 172.93.109.202:7777)
# Revision
6dc31db4f90e993d484e8afe3403f3c475506173

# Description
Upon examining myself by clicking on my sprite I was greeted to a notification telling me that the status of my legs are "OK", regardless of where I was targeting on my sprite.

# Steps to Reproduce
Click your little man while playing to examine him and if your legs are okay, you should too enjoy this message.

# What you Expected
I assumed that if I was targeting my arms, for example it would inform me if they were also okay.

# What Actually Happened
Clicked my sprite, my legs are okay at least.

### PR DESCRIPTION
**Pull request by MadmanMartian**
closes #20266

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -200,7 +200,7 @@
 					num_injuries++
 
 			if(num_injuries == 0)
-				to_chat(src, "My legs are OK.")
+				to_chat(src, "My limbs are okay.")
```
Comment (Exxion) [inline code/modules/mob/living/carbon/carbon.dm]: >okay

Comment (MadmanMartian) [reply code/modules/mob/living/carbon/carbon.dm]: no_difference_between_ok_and_okay

Comment (SonixApache) [reply code/modules/mob/living/carbon/carbon.dm]: REMOVE ISLANDER remove islander you are worst islander

Comment (Exxion) [reply code/modules/mob/living/carbon/carbon.dm]: I was joking and don't actually give a shit but [the spelling "OK" predates the spelling "okay" by centuries](https://books.google.com/ngrams/graph?content=ok%2Cokay&case_insensitive=on&year_start=0&year_end=9999&corpus=15&smoothing=3&share=&direct_url=t4%3B%2Cok%3B%2Cc0%3B%2Cs0%3B%3BOK%3B%2Cc0%3B%3Bok%3B%2Cc0%3B%3BOk%3B%2Cc0%3B%3BoK%3B%2Cc0%3B.t4%3B%2Cokay%3B%2Cc0%3B%2Cs0%3B%3Bokay%3B%2Cc0%3B%3BOkay%3B%2Cc0)

**Comment by MadmanMartian** [comment]
@DrSnips @jknpj @tolan1 @SonixApache You happy now?

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
The in-game spelling is OK

Anyways thanks I was going to make this PR but got distracted

**Comment by DrSnips** [comment]
My legs are ok now.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Super !!COMPROMISE!! time:
It shows "my body looks okay" or some gay shit normally
But it shows "My legs are OK." if hallucinating

**Comment by MadmanMartian** [comment]
That does sound more fun.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
:ok_hand: good shit

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -200,7 +200,10 @@
 					num_injuries++
 
 			if(num_injuries == 0)
-				to_chat(src, "My legs are OK.")
+				if(hallucinating())
+					to_chat(src, "<span class = 'orange'>My legs are [pick("okay","OK")].</span>")
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/modules/mob/living/carbon/carbon.dm]: Don't mean to sound picky, but the thing it's referencing is always OK, never okay

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
I'm sure the people that initially voted in this PR will come back and reconsider their votes after the changes

**Comment by MadmanMartian** [comment]
From experience, they don't.

**Pull request merged**